cxx-qt-lib/src/lib.rs

```diff
@@ -49,6 +49,66 @@ pub use qvariant::{QVariant, Variant, VariantValue};
 mod map_qt_value;
 pub use map_qt_value::*;
 
+pub const QT_TYPES_HEADER: &str = include_str!("../include/qt_types.h");
+pub const QT_TYPES_SOURCE: &str = include_str!("qt_types.cpp");
+
+pub const QCOLOR_CXX_HEADER: &str =
+    include_str!(concat!(env!("OUT_DIR"), "/cxx-qt-lib/include/qcolor_cxx.h"));
```

Comment:
Hopefully Windows is able to pick up these correctly as well, even if there is a mix of \ and /. Because we are only on the Rust side and not CMake will this be OK ? @Be-ing

Reply:
Seems this isn't, so we should use cfg to set the right path depending on the platform. https://github.com/rust-lang/rust/issues/75075 https://doc.rust-lang.org/stable/core/macro.include_str.html

Reply:
It sure would be nice if there was a way to tell Cargo to replace \ with / in all its environment variables that represent filesystem paths.